Lib/enum.py

```diff
@@ -539,42 +697,54 @@ def __setattr__(cls, name, value):
             raise AttributeError('Cannot reassign members.')
         super().__setattr__(name, value)
 
-    def _create_(cls, class_name, names, *, module=None, qualname=None, type=None, start=1):
+    def _create_(
+                cls, class_name, names,
+                *,
+                module=None, qualname=None, type=None,
+                start=1, boundary=None,
+        ):
         """
         Convenience method to create a new Enum class.
 
         `names` can be:
 
         * A string containing member names, separated either with spaces or
           commas.  Values are incremented by 1 from `start`.
-        * An iterable of member names.  Values are incremented by 1 from `start`.
+        * An iterable of member names.  Values are incremented by 1 from
+          `start`.
         * An iterable of (member name, value) pairs.
         * A mapping of member name -> value pairs.
         """
         metacls = cls.__class__
         bases = (cls, ) if type is None else (type, cls)
         _, first_enum = cls._get_mixins_(cls, bases)
         classdict = metacls.__prepare__(class_name, bases)
-
+        #
         # special processing needed for names?
         if isinstance(names, str):
             names = names.replace(',', ' ').split()
-        if isinstance(names, (tuple, list)) and names and isinstance(names[0], str):
+        if (
+                isinstance(names, (tuple, list))
+                and names
+                and isinstance(names[0], str)
+            ):
```

Comment:
Would you consider making formatting changes in a separate PR?  There is a lot to carefully review here and these can be distracting.

Reply:
Sorry, part of that is supposed to make it easier to read, and it's now a few commits back.  I don't have that much bandwidth available.  :(

Reply:
there's also the bandwidth of all the people that will review this PR to consider-- and the PR should be set up to get good reviews out of them (https://mtlynch.io/code-review-love/#6-separate-functional-and-non-functional-changes)

about 25 of the edits in enum.py are formatting changes-- it would help a lot if they weren't mixed with the code changes that need careful review

Reply:
You are, of course, correct.  In the future I will endeavor to keep them separate.